be1-go/db/sqlite_test.go

```diff
@@ -0,0 +1,183 @@
+package db
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"student20_pop/message"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSQLite_NewRepository(t *testing.T) {
+	repo, err := createRepo("test.db")
+	require.NoError(t, err)
+
+	err = repo.Close()
+	require.NoError(t, err)
```

Comment:
this could nearly be satisfied by two functions that merely return nil as an error.
should we at least check that the test.db file gets created?